### DISCUSSION TITLE
How to integrate @walletconnect/modal-react-native with wagmi ?

### DISCUSSION THREAD

**Discussion by mickael-btc** [WalletConnect/walletconnect-monorepo#3727] [Developer Support] [answered]
Hello,
I am working on a react native app. I saw a lot of examples with wagmi and wallet connect intgration but they seem to only work for web. I can't manage to make it work with wagmi. Any tip or simple example would be very helpful !

**Comment by naresh-bls** [comment]
Hi @WalletConnect Team, 
@ignaciosantise @crypblizz8 @Mostafatalaat770 @m1heng
I just want to know if any update on this.

**Reply by glitch-txs** [reply, marked as answer]
Hi, we're currently working on a React Native + Viem implementation, we are considering using Wagmi as well but this is TBA yet.
